lib/browser/guest-window-manager.js

```diff
@@ -242,10 +242,14 @@ ipcMain.on('ELECTRON_GUEST_WINDOW_MANAGER_INTERNAL_WINDOW_OPEN', function (event
                                                                   disposition, options,
                                                                   additionalFeatures, postData) {
   options = mergeBrowserWindowOptions(event.sender, options)
+  const {webContents} = options
```

Comment:
It is better to use `options.webContents` directly, a single `webContents` can have lots of meanings and it is hard to find its declaration when reading the code.